runner/output_t.go

```diff
@@ -0,0 +1,59 @@
+package runner
+
+import (
+	"io/ioutil"
+	"net/http"
+	"strings"
+
+	"github.com/coduno/app/util"
+	"github.com/coduno/piper/docker"
+)
+
+// OutputTestHandler is the handler for a run with a simple output check
+type OutputTestHandler struct {
+	TestFilePath string
+}
+
+// Handle function for an output test run. It writes the file with code in
+// the tmp folder and  returns the docker run configuration.
+func (oth OutputTestHandler) Handle(w http.ResponseWriter, r *http.Request) (c docker.Config) {
+	return GeneralHandle(w, r)
+}
+
+// Respond implementation for an output test run. It returns the number of
+// different lines between the user output, the user output and the test
+// output.
+func (oth OutputTestHandler) Respond(w http.ResponseWriter, req *http.Request, res docker.Result) {
+	buf, err := ioutil.ReadFile(oth.TestFilePath)
+	if err != nil {
+		http.Error(w, err.Error(), http.StatusInternalServerError)
+		return
+	}
+	test := strings.Split(string(buf), "\n")
```

Comment:
I think splitting the output inside the generic code is a bit too much. We should leave that to the specific evaluation part.